src/service/mod.rs

```diff
@@ -0,0 +1,152 @@
+use rocket::async_trait;
+
+pub mod name;
+pub mod s3;
+
+#[async_trait]
+pub trait KeplerService {
+    type Error;
+    type Stopped;
```

Comment:
I don't understand why it's called `Stopped`

Reply:
The idea was to indicate a type which represents the service without the associated tasks running, so the service is `Stopped`. This would let a "manager" type hold a set of `dyn KeplerService` which can be started and stopped using a uniform interface, but this is maybe premature/YAGNI